relayer/chains/substrate/event_parser.go

```diff
@@ -145,7 +151,7 @@ func (scp *SubstrateChainProcessor) parseEvent(
 // returns the unique key for packet accumulator cache
 func genAccumKey(data interface{}) ibcPacketKey {
 	return ibcPacketKey{
-		sequence:   data.(ibcEventQueryItem)["sequence"].(uint64),
+		sequence:   cast.ToUint64(data.(ibcEventQueryItem)["sequence"].(float64)),
```

Comment:
It looks like we are using type assertions to go from a string to a float64 and then casting to a uint64, this feels a little clunky.

I believe packet sequence numbers are of type `uint64` in IBC. Is there any reason why we can't handle this value as a string or at least parse/type assert right into a `uint64`?

Reply:
The map is being converted from json package of golang, every number is being detected as float64